csrc/scheduler/transpose.cpp

```diff
@@ -569,6 +592,26 @@ std::string getTransposeRuntimeRejectReason(
   }
 #endif
 
+  // disallow transpose scheduler when we have a combination of:
+  // 1. view op; and
+  // 2. small transpose transformation
+  // See note [Supporting small transpose dimensions]
+  const auto index_type = runtime_info.getIndexType();
+  auto params =
+      std::make_shared<TransposeParams>("Transpose heuristics", index_type);
+  maybeBuildVirtualInnerDims(
+      *params,
+      device_multiprocessor_count,
+      n_elems,
+      shape_in_ref1,
+      inner_most_pos1_in_ref1,
+      inner_most_pos2_in_ref1);
+
+  if (!scheduler_utils::getViewTVs(fusion).empty() &&
+      hasSmallTransposeDimensions(params)) {
+    return "Small transpose dimensions and view op cannot be currently be handled by transpose scheduler. See: https://github.com/NVIDIA/Fuser/pull/592";
```

Comment:
added this part to reject view + small transpose dimensions per @zasdfgbnm 's suggestion in our chats.